index.dd

```diff
@@ -111,6 +111,18 @@ void main()
 }
 ----
 )
+$(EXTRA_EXAMPLE Invoke external programs,
+----
+void main()
+{
+    import std.exception, std.stdio, std.process;
+
+    auto result = ["whoami"].execute;
+    enforce(result.status == 0);
+    result.output.write; // so, who are you?
```

Comment:
I think the comment is a bit over the top :) Someone non-technical might get confused or insulted (it happens).